pkgs/servers/web-apps/searx/default.nix

```diff
@@ -14,6 +14,11 @@ buildPythonApplication rec {
     sha256 = "0hfa4nmis98yvghxw866rzjpmhb2ln8l6l8g9yx4m79b2lk76xcs";
   };
 
+  patches = [(fetchpatch {
+    url = "https://github.com/asciimoo/searx/commit/b8b13372c8fd3bfe978a1c724ab98b05348df054.patch";
+    sha256 = "0wrcywxihxiz1dka3mjbzprv8qznan0x4b0qjn0zfd8167pdwnk2";
```

Comment:
Please note that fetchpatch is not compatible with hashes returned by `nix-prefetch-url`.

Reply:
noted for the next time and fixed